e2e/wdio/headless/test.e2e.ts

```diff
@@ -107,8 +107,7 @@ describe('main suite 1', () => {
         ]
 
         before(async () => {
-            await browser.url('http://guinea-pig.webdriver.io/')
-            await browser.$('a[href="pointer.html"]').click()
```

Comment:
Why was the click removed? This potentionally affects other tests

Reply:
The suite of test related to only moveTo tests and it's been changed directly to http://guinea-pig.webdriver.io/pointer.html' not necessary click it here, the rest cases call browser.url with their own links and open it each time when it's been called